### PR TITLE
VAGOV-3888: admin menu improvements. (Sprint 16)

### PR DESCRIPTION
**Pull request by kevwalsh**
QA notes: https://va-gov.atlassian.net/browse/VAGOV-3888

### CODE REVIEW

**Comment by kevwalsh** [comment]
rebuild

**Comment by andyhawks** [comment]
rebuild

**Comment by ElijahLynn** [comment]
Failed because the composer.lock file was not updated. Best to always run `composer require ...` to add packages.

**Comment by ElijahLynn** [comment]
Sorry @kevwalsh merged in some composer changes before this. I'll fix the conflict for you on Monday.

**Comment by kevwalsh** [comment]
@ElijahLynn is this stil on your radar for today? if not i'll try to get to it in the morning.

File: tests/phpunit/SecurityRolesPermissionsTest.php [hunk 1]
```diff
@@ -465,8 +465,8 @@ public function expectedPerms() {
           'manage password reset',
           'use moderation dashboard',
           'use text format rich_text',
-          'view sections in toolbar',
           'view any moderation dashboard',
+          'view sections in toolbar',
```
Comment (ElijahLynn) [inline tests/phpunit/SecurityRolesPermissionsTest.php]: Can we clarify this here causing tests to pass? I am seeing that the ordering matters here or am I just not seeing the change here besides ordering?

Comment (kevwalsh) [reply tests/phpunit/SecurityRolesPermissionsTest.php]: @ElijahLynn  seems like it's just an ordering problem yeah.

Comment (kevwalsh) [reply tests/phpunit/SecurityRolesPermissionsTest.php]: i thought i had the same order originally, but i was comparing manually. the comparison is between an array that has line breaks, and one that does not.

Comment (ElijahLynn) [reply tests/phpunit/SecurityRolesPermissionsTest.php]: thanks

**Pull request merged**